### ISSUE TITLE
Error application 

### ISSUE THREAD

**Issue by AYMENJD** [melchor629/youtubedl-audio-api#1]
It's not working on heroku error application

**Comment by melchor629** [comment]
Hello, I need more detailed information about what is the error.

- Are you deploying using _Heroku GIt_, _Github integration_ or _Container Registry_?
- Did you check the logs? What are they saying?

I will check if I see something wrong on the Heroku deployment...

**Comment by melchor629** [comment]
Note: I found no error is written to the logs using _Heroku Git_ deployment method.

**Comment by AYMENJD** [comment]
> Note: I found no error is written to the logs using _Heroku Git_ deployment method.

In the logs i get this error : 
2020-05-03T10:12:18.243195+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=yt-audio-ar.herokuapp.com request_id=83580c56-bc20-44cd-aade-b044e832d1fd fwd="52.16.87.7" dyno= connect= service= status=503 bytes= protocol=https
2020-05-03T10:12:18.937205+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=yt-audio-ar.herokuapp.com request_id=c9c1abd2-1f7e-4c8e-af35-333ad29c173b fwd="52.16.87.7" dyno= connect= service= status=503 bytes= protocol=https

+ I use github to deploy i forked your repo and deploy it on heroku + even your web is not working https://yt-audio-api.herokuapp.com/api/0RLvtm0EghQ

**Comment by melchor629** [comment]
I think I now what is happening and could happen elsewhere (not only on Heroku).

As you can see, logs do not say anything useful, just it crashed, which makes this debug harder.

My app is not on Heroku anymore but I did not changed the URLs yet :(

**Comment by melchor629** [comment]
Ok so I managed to deploy properly in Heroku with the changes I just pushed into this repository. The issue was that I had the gunicorn configuration in a file called `gunicorn.py` and this caused the whole `gunicorn` package to "disappear". So it could not start the app in there. Just renamed the file and now it seems to work.

Test it on your own environment and respond the issue if there is any other issue.

Note that I added a **Note** in the README about workers. After deploying the app, go to the settings, Config Vars and put something like this. By default uses to much workers for Heroku which can cause the app to be killed by Heroku itself due to high memory usage:
![Captura de pantalla 2020-05-03 a las 12 31 58](https://user-images.githubusercontent.com/1056963/80911968-19e0bd80-8d3a-11ea-9d79-ee5546ea4908.png)

Thanks for creating this issue :)
